sql/src/main/scala/org/apache/spark/sql/sedona_sql/expressions/Functions.scala

```diff
@@ -1782,3 +1782,26 @@ case class ST_XMin(inputExpressions: Seq[Expression])
     copy(inputExpressions = newChildren)
   }
 }
+
+
+/**
+ * Returns the areal geometry formed by the constituent linework of the input geometry assuming assumes all inner geometries represent holes
+ *
+ * @param inputExpressions
+ */
+case class ST_BuildArea(inputExpressions: Seq[Expression])
+  extends UnaryGeometryExpression with CodegenFallback {
+  assert(inputExpressions.length == 1)
+
+  override protected def nullSafeEval(geometry: Geometry): Any = {
```

Comment:
Should this be a nullable function?

Reply:
Yes, this should be a nullable function. So no need to extend UnaryGeometryExpression

Reply:
Have updated the function and added null checks